tests/test_jinja2.py

```diff
@@ -158,3 +159,16 @@ def test_inventory_global_context(self):
             context = {"inventory_global": inv_global}
             f.seek(0)
             self.assertEqual(render_jinja2_file(f.name, context), cluster_name)
+
+
+class Jinja2ExternalFilterTest(unittest.TestCase):
+    def test_custom_filter_jinja2(self):
+        with tempfile.NamedTemporaryFile() as f:
+            f.write("{{ inventory.parameters.cluster.name | custom_filter }}".encode("UTF-8"))
+            cluster_name = "minikube"
+            target_name = "minikube-es"
+            inv = inventory(["examples/kubernetes"], target_name)
+            context = {"inventory": inv}
+            f.seek(0)
+            self.assertEqual(render_jinja2_file(f.name, context, ["./examples/kubernetes/lib/custom_filter.py"]), base64_encode(cluster_name))
```

Comment:
Move the `render_jinja2_file(f.name, context, ["./examples/kubernetes/lib/custom_filter.py"]),` and `base64_encode(cluster_name)` to separate lines and add to variables like `expected_output` and `actual_output`. 
Try keeping your code in <100 characters of width. It improved readability for people using smaller screens. :+1: